dev/src/index.ts

```diff
@@ -580,14 +580,21 @@ export class Firestore implements firestore.Firestore {
           const grpcModule = this._settings.grpc ?? require('google-gax').grpc;
           const sslCreds = grpcModule.credentials.createInsecure();
 
-          client = new module.exports.v1(
-            {
-              sslCreds,
-              ...this._settings,
-              fallback: useFallback,
-            },
-            gax
-          );
+          // eslint-disable-next-line @typescript-eslint/no-explicit-any
+          const settings: ClientOptions = {
+            sslCreds,
+            ...this._settings,
+            fallback: useFallback,
+          };
+
+          // If `ssl === false` and we're using the GAX fallback,
```

Comment:
nit: Can this comment be collapsed into 2 lines (instead of 3) without exceeding the max line length?

Reply:
Thanks.